### DISCUSSION TITLE
Cannot acces ip inside a docker container

### DISCUSSION THREAD

**Discussion by Attacler** [QuorumDMS/ftp-srv#268] [Q&A]
Hello there!

So basicly i got a docker image which i would like to use ftp-srv in.
But at the moment if i create the container, this happends:
![image](https://user-images.githubusercontent.com/42644889/120523858-b7437080-c3d6-11eb-99dc-1479c33c4df1.png)

I can however make the ip go to 127.0.0.1 but then the ftp server closes the connection.

What is the best way to fix this?
(if i host a http server on the same port, it does work so the port is reachable)

### Related discussions:
#278

**Comment by matt-forster** [comment]
This is a great question - but I am going to convert this to a discussion, as that is where this belongs!

**Comment by mcfriend99** [comment]
@Attacler were you ever able to find a solution??
